pom.xml

```diff
@@ -55,6 +55,7 @@
     <modules>
         <module>timachine-core</module>
         <module>timachine-maven-plugin</module>
+        <module>timachine-maven-plugin-test</module>
```

Comment:
@KengoTODA  I intended not to add `timachine-maven-plugin-test` module.

That module is for manually testing the plugin usage through command line, since I didn't use unit test for the plugin. So I don't want it to be build normally.

If you have some good way to test plugin usage automatically, please tell me, thanks!